BUILD.md

```diff
@@ -98,10 +98,16 @@ process. For instance:
     cd debug/src/unity
     make -j 4
 
+If `Ninja` is chosen as the generator in the previous step, you should use command listed below instead:
+
+    cd debug/
+    # check ninja -h for more info
+    ninja -j 4
```

Comment:
We should recommend running `ninja` with no arguments, since it automatically parallelizes based on the number of CPU cores you have. I only use `-j` if I want to artificially limit it.

Reply:
Just want to align with the `make` example. I can remove it if it's redundant.

Reply:
I think we could make it `ninja` (without `-j 4`), and add a comment that says ninja will automatically parallelize by default.

Reply:
I just notice you have the notice to recommend user use default parallelism. A new commit is made.